src/scrabble.js

```diff
@@ -1,6 +1,73 @@
 class Scrabble {
   // Write your implementation here
+  constructor(word) {
+    this.word = word;
+    this.points = 0;
+  }
 
-}
+  score(word) {
+    if (this.word === "" || this.word === " ") {
+      this.points += 0;
+      return 0;
+    }
+
+    if (this.word === " \t\n") {
+      this.points += 0;
+      return 0;
+    }
+
+    if (this.word === null) {
+      this.points += 0;
+      return 0;
+    }
```

Comment:
could be replaced with if (this.word === " " || this.word === " \t\n" || this.word === null){return 0}

not a big deal, but can condense the code

Reply:
Also, what if this.word is other data types (e.g NaN, false etc)? It might pass the conditionals...
I would suggest doing `if(typeof this.word !== "string")`